patch/series

```diff
@@ -253,6 +253,7 @@ armhf_secondary_boot_online.patch
 0028-EDAC-elba-Add-Elba-EDAC-support.patch
 0029-arm64-traps-Handle-SError-interrupt.patch
 0030-quirks-for-the-Pensando-qspi-controller.patch
+0031-platform-pensando-enable-ionic-x86.patch
```

Comment:
You need to use `patch/kconfig-inclusions` to set/enable kconfig options.

Reply:
Done